EasyPost/Client.cs

```diff
@@ -108,5 +112,37 @@ public Client(ClientConfiguration clientConfiguration)
 
             _dotNetVersion = dotNetVersion;
         }
+
+        /// <summary>
+        /// Return the request time from the client object.
+        /// </summary>
+        /// <returns> connection timeout in milliseconds.</returns>
+        public int getConnectionTimeout(){
+            return _connectTimeoutMilliseconds;
```

Comment:
```suggestion
return _connectTimeoutMillseconds ?: _defaultTimeoutMilliseconds;
```

Reply:
Oops, that's Groovy syntax, my bad. You'll have to add _connectTimeoutMilliseconds in between `?` and `:`

Reply:
+1 this is along the lines of what I was thinking of. We need to retain the `default` variables and keep them internal and unchanging. We also need to have a variable that's public that users can set to override the default values for timeouts.

Reply:
I'm thinking we can do this with some fancy getter/setter magic.

Reply:
I was following how our other library is set up like PHP [here](https://github.com/EasyPost/easypost-php/blob/master/lib/EasyPost/EasyPost.php#L148) and try to avoid additional variables. I will make some changes shortly and add an unit test